### DISCUSSION TITLE
Discord ban appeal

### DISCUSSION THREAD

**Discussion by ghost** [Pryaxis/TShock#2327] [Q&A] [answered]
It looks like i m banned in pryaxis discord server so i would know how can i appeal the ban

**Comment by hakusaro** [comment, marked as answer]
Our standard unban policy is to submit/provide/post a 750 word essay on why you should be unbanned.

**Reply by hakusaro** [reply]
It looks like you were banned because you attempted to send a message to all members on the server several times, after waiting only ten minutes. Because I didn't implement this ban, I can't unban you directly.

<img width="513" alt="Screen Shot 2021-05-22 at 11 46 46" src="https://user-images.githubusercontent.com/532647/119237810-a13ed180-baf3-11eb-92e7-bf9e685f1ef0.png">

**Reply by ghost** [reply]
But writing 750 words just to get reason for unban it.s stupid what shouĺd i write in it using 750 words?

**Reply by hakusaro** [reply]
> But writing 750 words just to get reason for unban it.s stupid what shouĺd i write in it using 750 words?

Your goal, to be straightforward, is to convince us that you care about the community, why we have the guidelines we do, and why you respect everyone in the community, and the time we spend working on it, and helping people.

You have not yet convinced us that you care about us.

**Reply by ghost** [reply]
I care about help that you can give.
